package/network/services/hostapd/files/hostapd.sh

```diff
@@ -376,6 +380,27 @@ hostapd_set_bss_options() {
 		append bss_conf "iapp_interface=$ifname" "$N"
 	}
 
+	json_get_vars ieee80211v
+	set_default ieee80211v 0
+	if [ "$ieee80211v" -eq "1" ]; then
+		json_get_vars time_advertisement time_zone wnm_sleep_mode bss_transition \
+			proxy_arp na_mcast_to_ucast
+
+		set_default time_advertisement 0
+		set_default wnm_sleep_mode 0
+		set_default bss_transition 0
+		set_default proxy_arp 0
+		set_default na_mcast_to_ucast 0
+
+		append bss_conf "time_advertisement=$time_advertisement" "$N"
+		[ -n "$time_zone" ] && append bss_conf "time_zone=$time_zone" "$N"
+
+		append bss_conf "wnm_sleep_mode=$wnm_sleep_mode" "$N"
+		append bss_conf "bss_transition=$bss_transition" "$N"
+		#append bss_conf "proxy_arp=$proxy_arp" "$N"
+		#append bss_conf "na_mcast_to_ucast=$na_mcast_to_ucast" "$N"
```

Comment:
If you're going to comment them out, please leave them out entirely.

Reply:
Ok, I commented them out because the hostapd version (2016-12-19) is not updated and those flags were not yet implemented in that versoin.
But I've just finished updating the hostapd package to the latest upstream commit on another branch and I think I'll make a PR about the hostapd update first and then I'll make the 80211v PR.

Reply:
Don't put too much effort in updating hostapd. I have an update queued in my staging dir for some weeks now, but it breaks build due to some conflicts. If you do want to make an update, please start from https://git.lede-project.org/?p=lede/stintel/staging.git;a=shortlog;h=refs/heads/hostapd

Reply:
What kind of conflicts?
I've fixed the patches errors using quilt and mine is compiling fine.
Where we can discuss about this?

Reply:
See http://lists.infradead.org/pipermail/lede-dev/2017-September/008788.html
You can reply to the thread, or come on IRC, or ... Either way, since I did not get any feedback, I'm probably going to merge those changes somewhere this week.

Reply:
```
since I did not get any feedback
```
Did you see this?
http://lists.infradead.org/pipermail/lede-dev/2017-September/008799.html

Reply:
Yes, you misunderstood. That's the patch that is in my staging tree. I did not get feedback about the 2 patches to fix the build problem.